viz/pkg/healthcheck/healthcheck.go

```diff
@@ -40,7 +46,19 @@ type HealthChecker struct {
 func NewHealthChecker(categoryIDs []healthcheck.CategoryID, options *healthcheck.Options) *HealthChecker {
 	parentHC := healthcheck.NewHealthChecker(categoryIDs, options)
 	hc := &HealthChecker{HealthChecker: parentHC}
-	parentHC.AppendCategories(*hc.vizCategory())
+
+	vizCheckMap := map[healthcheck.CategoryID]healthcheck.Category{}
```

Comment:
👍  on this suggestion. I think, imo, it would improve readability if we explicitly pass in the actual categories we'd like to check for. My one comment is that we need to ensure all other sub checks follow this pattern as well. Definitely for some other PR down the line.

Reply:
Updated. But the current state is also a bit confusing for the following reason
- Now, specifying Categories for the viz hc are configured in two different ways. One way to specify the core cp checks i.e through the CategoryIDs argument and another way is for the Viz checks by directly adding them using `hc.AppendCategories`